common/common-api/src/main/java/org/ow2/proactive/utils/ObjectByteConverter.java

```diff
@@ -70,6 +75,9 @@
     public static final byte[] objectToByteArray(Object obj, boolean compress) throws IOException {
         ByteArrayOutputStream baos = null;
         ObjectOutputStream oos = null;
+        if (obj == null) {
```

Comment:
We want to be able to convert a null result object altogether. In that case, it should not be transformed as a empty byte array, but keep the null quality. The result can be null, if for example an exception occurred during the task